kolibri/core/auth/test/test_morango_integration.py

```diff
@@ -132,14 +134,24 @@ class EcosystemTestCase(TestCase):
 
```

Comment:
Most of the rest of this file (except where otherwise noted) is just cleanup and refactors of the existing tests, to use the helper methods and some less verbose idioms.